engine/src/ast/field_expr.rs

```diff
@@ -14,7 +14,7 @@ use fnv::FnvBuildHasher;
 use indexmap::IndexSet;
 use memmem::Searcher;
 use serde::{Serialize, Serializer};
-use std::{cmp::Ordering, net::IpAddr};
+use std::{cmp::Ordering, convert::Into, net::IpAddr};
```

Comment:
You don't need to import `convert::Into`, it's in scope by default.

Reply:
Fixed!